Add drag-and-drop config and update shift-click to support unequip

Memebeams opened the pull request:
* Shift-click feature reworked to get around feature not working for unequip and when clicking fragment icons
* Drag-and-drop modes to increase anti-drag behavior or disable drag and drop entirely (added because it's kind of hard to quickly shift-click multiple fragments quickly with the vanilla drag)
* Funny typo in config fixed
  * `shitClickEquipFragment` -> `shiftClickToggleFragment`

Memebeams committed:
Add drag-and-drop config and update shift-click to support unequip

tylerthardy commented:
Shift equip & unequip don't appear to work for the following:
Hot on the Trail
Dragon On a Bit

I'm not sure if it's introduced by this PR, but the unequip is affected.

SyntaxBlitz commented:
probably a capitalization problem, I figure. I'll have a look

SyntaxBlitz commented on src/main/java/io/tja/osrs/shattered_relics_fragment_presets/ShatteredRelicsFragmentPresetsConfig.java:
```diff
@@ -36,13 +36,26 @@ default int resizableOffsetY() {
     )
-    default boolean shitClickEquipFragment() {
+    default boolean shiftClickToggleFragment() {
         return false;
```
I think I'm going to turn this on by default. only reason we didn't do that before is that it wasn't working very well

SyntaxBlitz committed:
Cleanup

- fix IMPROVED drag-drop (missing a break statement)
- turn on shift-click by default
- use Text.removeTags instead of regex
- just convert everything lowercase, because I cba to verify all of the fragment
titles. this plugin will be irrelevant in a couple weeks anyway
- remove Set.of because that's java 1.9+ and we target 1.8 for runelite
plugins

SyntaxBlitz commented:
@Memebeams are you able to give me push access to the branch? there are a few changes I need to make but I don't think I can push them without making a fresh PR

Memebeams committed:
Make shift-click on by default and fix capitalization for Dragon On a Bit and

Memebeams commented:
> @Memebeams are you able to give me push access to the branch? there are a few changes I need to make but I don't think I can push them without making a fresh PR

@SyntaxBlitz 
I've pushed a commit with the two improvements you described above, but GitHub shows this setting on my screen:
![image](https://user-images.githubusercontent.com/5132037/152479604-9f6d8a22-e627-4653-867e-582779116f24.png)

So you should be able to commit to my branch if you need to. If not I'll give you write access.

SyntaxBlitz committed:
Merge branch 'master' of https://github.com/Memebeams/osrs-shattered-relics-fragment-presets into Memebeams/master

SyntaxBlitz commented:
ah, I think I might have just done it wrong before. Thanks! here are the changes I made:

- fix `IMPROVED` drag-drop (missing a break statement)
- turn on shift-click by default
- use Text.removeTags instead of regex
- just convert everything to lowercase, because I cba to double-check all of the fragment titles. this plugin will be irrelevant in a couple weeks anyway so it's ok if the code is a little crappy
- remove Set.of because that's java 1.9+ and we target 1.8 for runelite plugins

SyntaxBlitz commented:
thanks for the contribution @Memebeams! and thanks for pointing out the capitalization issue @tylerthardy. I'm excited to merge this! it's going to be really useful.
